src/libraries/System.Security.Cryptography/tests/X509Certificates/CollectionTests.cs

```diff
@@ -637,7 +637,7 @@ public static void ImportFromFileTests(X509KeyStorageFlags storageFlags)
         [Fact]
         public static void ImportMultiplePrivateKeysPfx()
         {
-            using (ImportedCollection ic = Cert.Import(TestData.MultiPrivateKeyPfx))
+            using (ImportedCollection ic = Cert.Import(TestData.MultiPrivateKeyPfx, (string?)null, X509KeyStorageFlags.DefaultKeySet))
```

Comment:
Maybe it would be better to just make `Cert.Import` pass these flags on its own, and if there's a test that wants the count to be enforced then make a new method (Cert.ImportWithIterationLimit, or whatever))?

Reply:
`Cert.Import` advices against making this kind of change:
https://github.com/dotnet/runtime/blob/1d9b36d3b6d23e37e9959ad85360b4041eeb90a9/src/libraries/System.Security.Cryptography/tests/X509Certificates/Cert.cs#L25-L27